src/OrchardCore.Modules/OrchardCore.Localization/PermissionProvider.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Localization;
+
+/// <summary>
+/// Represents the localization module permissions.
+/// </summary>
+public class PermissionProvider : IPermissionProvider
+{
+    public PermissionProvider(IStringLocalizer<PermissionProvider> S)
+    {
+        Permissions.ManageCultures = new Permission(nameof(Permissions.ManageCultures), S["View audit trail"]);
```

Comment:
@sebastienros If this style is acceptable, I will loop through every permissions in the code-bae